src/libs/ActiveClientManager/types.ts

```diff
@@ -0,0 +1,7 @@
+type ActiveClientManagerModule = {
```

Comment:
Expanding more on my first comment about not changing the exports, you can create separate types for each function and use it in the files.

```ts
type Init = () => void;
type IsClientTheLeader = () => boolean;
...
```